signal_scanner_bot/env.py

```diff
@@ -13,8 +14,43 @@
 class _State:
     """Class for holding global state across threads/tasks"""
 
-    LISTENING = False
-    STOP_REQUESTED = False
+    # Initialize state, set LISTENING to true if state file exists,
+    # sets false if not.
+    def __init__(self, file: str):
+        self.file = file
+        self.LISTENING = True if os.path.isfile(file) else False
+        self.STOP_REQUESTED = False
+
+    # Method to update the listening status of the State class
+    # object. Checks for on/off and creates/deletes state file.
+    def update_listening_status(self, status: str) -> Optional[str]:
+        if status == "AUTOSCANON":
+            self.LISTENING = True
+            open(self.file, "a").close()
+            return "==Auto Scanning Activated=="
+        elif status == "AUTOSCANOFF":
+            self.LISTENING = False
+            try:
+                os.remove(self.file)
+            except OSError:
+                pass
```

Comment:
With a `Path` object:
```suggestion
            self.file.unlink(missing_ok=True)
```